strictdoc/export/excel/excel_generator.py

```diff
@@ -1,59 +1,193 @@
 import os
 from pathlib import Path
+from typing import Dict, List
 
 import xlsxwriter
+from xlsxwriter.workbook import Workbook
+from xlsxwriter.worksheet import Worksheet
+from xlsxwriter.utility import xl_cell_to_rowcol_abs
 
 from strictdoc.backend.dsl.models.document import Document
+from strictdoc.backend.dsl.models.requirement import Requirement
 from strictdoc.core.document_iterator import DocumentCachingIterator
 from strictdoc.core.document_tree import DocumentTree
 
 
-def get_path_components(folder_path):
-    path = os.path.normpath(folder_path)
-    return path.split(os.sep)
+EXCEL_SHEET_NAME = "requierements"
+MAX_WIDTH = 75
+HEADER_MARGIN = 3
+MAX_WIDTH_KEY = "max_width"
+COLUMN_HEADER_KEY = "header"
+PARENT_COLUMN_HEADER_LABEL = "PARENT"
+EXPORT_COLUMNS = [
+    {
+        "name": "uid",
+        "header": "UID",
+    },
+    # {
+    #     "name": "title",
+    #     "header": "TITLE",
+    # },
+    {
+        "name": "statement",
+        "header": "STATEMENT",
+    },
+    # {
+    #     "name": "rationale",
+    #     "header": "RATIONALE",
+    # },
+    # {
+    #     "name": "body",
+    #     "header": "BODY",
+    # },
+]
 
 
 class ExcelGenerator:
+    @staticmethod
+    def lookup_refs(document_contents: List) -> Dict[str, int]:
```

Comment:
Please move all non-public methods down so that the `export_tree` is at the top of the class.